src/Raven.Server/Documents/Expiration/ExpirationStorage.cs

```diff
@@ -246,98 +235,146 @@ private static bool HasPassed(BlittableJsonReaderObject metadata, string metadat
             return false;
         }
 
-        public int DeleteDocumentsExpiration(DocumentsOperationContext context, Dictionary<Slice, List<(Slice LowerId, string Id)>> expired, DateTime currentTime)
+        public class ExpiredDocumentInfo
```

Comment:
`Expired` may mislead that the document has already expired. 
We should name this class `DocumentExpirationInfo`, `ScheduledExpirationDocument`, or similarly.